### DISCUSSION TITLE
Login with default management account gives Not Authorized (Fresh docker installation)

### DISCUSSION THREAD

**Discussion by MatthijsVollenbroek** [rabbitmq/rabbitmq-server#10288] [Other] [answered]
### Describe the bug

I am trying to set up a new docker container of rabbit mq, but sadly it is not working as I was expecting. As I am unable to log into the management page.

### Reproduction steps

1. First, i ran this command `docker run -d --hostname rmq --name rabbitmq -p 15672:15672 -p 5672:5672 rabbitmq:3.12-management`
2. Then, I went to the rabbitmq management console and use user guest password guest to login
3.  As I submit this, i get a browser login popup: 
![image](https://github.com/rabbitmq/rabbitmq-server/assets/46601320/f30529f1-5b59-4823-bc12-1648c752bfc2)
4. If I enter the credentials again, the popup just keeps re-appearing
5. Once I hit cancel, I get a 401 error: 
![image](https://github.com/rabbitmq/rabbitmq-server/assets/46601320/853dd77b-3ebc-4405-aefd-1073f74a9d56)
...

### Expected behavior

I was expecting it to log me into the management console, maybe i am doing something wrong?
Searching on the internet and watching various tutorials, I am unable to find a solution that works for me.

Thanks in advance for helping me out!

### Additional context

I already tried adding a new user using:
```
rabbitmqctl add_user admin admin
rabbitmqctl set_user_tags admin administrator
rabbitmqctl set_permissions -p / admin ".*" ".*" ".*"
```
And also adding both ports to the allowed connections in the windows firewall, without any success.

**Comment by michaelklishin** [comment, marked as answer]
Please use GitHub Discussions for questions, they have been around for a good couple of years and no longer an exotic unknown feature.

You are trying to log into a remote node using default user's credentials. That is [explicitly prohibited](https://rabbitmq.com/access-control.html#loopback-users) for fairly obvious security reasons.

Your CLI commands do not specify any node name. That's usually fine for nodes running locally (in the host) but again, your node is running in a container, it is a separate host. So I am not sure if your command actually reach the node you think they reach.

See node logs for clues. When a new user is added, the node that does it logs a very specific message.

**Comment by MatthijsVollenbroek** [comment]
Hi Michael,

Thank you very much for the quick reply, the issue is solved! (See update).

Sorry for creating a bug, I will be more mindful of that in the future!

I was checking the logs of the RabbitMQ container and I see these logs after executing the cli commands:
```
2024-01-08 02:31:48 2024-01-08 01:31:48.171345+00:00 [info] <0.799.0> Created user 'test'
2024-01-08 02:32:20 2024-01-08 01:32:20.924237+00:00 [info] <0.814.0> Successfully set user tags for user 'test' to [administrator]
2024-01-08 02:33:02 2024-01-08 01:33:02.137727+00:00 [info] <0.825.0> Successfully set permissions for user 'test' in virtual host '/' to '.*', '.*', '.*'
```
As per the documentation, creating this new "test" user should allow me to sign in into the RabbitMQ, regardless whether it is localhost or not. Still, I was facing the same problem when trying to login with the newly created Test User.

**Update:** rebooting my pc somehow fixed the issue, classic! I guess my browser was still somehow logged into an older version with the guest account.

**Reply by michaelklishin** [reply]
It is possible that you had to clear a browser cache. I have forgotten to mention it, sorry.

**Comment by lukebakken** [comment]
For what it's worth, I can't reproduce the reported issue. Here is the output from starting the docker image on a Ubuntu 22 VM:

```
lbakken@PROKOFIEV ~
$ docker run -d --hostname rmq --name rabbitmq -p 15672:15672 -p 5672:5672 rabbitmq:3.12-management
Unable to find image 'rabbitmq:3.12-management' locally
3.12-management: Pulling from library/rabbitmq
a48641193673: Already exists
0c6774f8669e: Pull complete
345b513f4387: Pull complete
d425d9b0a570: Pull complete
5ad710591657: Pull complete
75b400e36267: Pull complete
5f9ef9027ea3: Pull complete
49fb96d8a408: Pull complete
276633c311d1: Pull complete
d85545149ba6: Pull complete
b248ca730a81: Pull complete
eae5924a2823: Pull complete
Digest: sha256:a44ba5a23397c3d84f8639f1adc037555541877df8fd370ef460133548e21b36
Status: Downloaded newer image for rabbitmq:3.12-management
f8e28d0c82185ac5c543234064c8b03f65d863872515aedc2d7d1c29cce5eaf4
lbakken@PROKOFIEV ~
$ docker ps
CONTAINER ID   IMAGE                      COMMAND                  CREATED          STATUS          PORTS
                                                                                             NAMES
f8e28d0c8218   rabbitmq:3.12-management   "docker-entrypoint.s…"   14 seconds ago   Up 13 seconds   4369/tcp, 5671/tcp, 0.0.0.0:5672->5672/tcp, :::5672->5672/tcp, 15671/tcp, 15691-15692/tcp, 25672/tcp, 0.0.0.0:15672->15672/tcp, :::15672->15672/tcp   rabbitmq
```

I then used Chrome to browse to `localhost:15672`, entered `guest` / `guest`, and was logged in as expected.

**Comment by lukebakken** [comment]
@MatthijsVollenbroek - questions:

* Do you have success if you try to log in using an Incognito (Private) browsing tab in Chrome?
* How are you running Docker on Windows? I use Windows 11 Pro, and WSL2 to run Ubuntu 22.04

**Reply by MatthijsVollenbroek** [reply]
Hey,

I found out the cause of the issue. The reason it worked after reboot, was that I still had a different rabbitmq container running as part of my other job, so it was trying to connect to that one. It has a cryptic name so I didn't notice it earlier. Mistake on my end!

Regardless, thanks for your efforts!
